### ISSUE TITLE
Header.rightItems - Error code: 1046: Type was not found or was not a compile-time constant: DisplayObject.

### ISSUE THREAD

**Issue by teotidev** [BowlerHatLLC/feathers-sdk#30]
Hey, I have been getting this error which obviously the compiler is not importing DisplayObject behind the scenes, you run into this?

If I add `import starling.display.DisplayObject;` the error goes away.

![capture](https://cloud.githubusercontent.com/assets/4727772/11615278/5c701156-9c2a-11e5-9e2d-dc5f30260ea6.JPG)

**Comment by joshtynjala** [comment]
Yes, I've noticed this issue. If I recall correctly, I could also reproduce it in the Flex SDK. For some reason, the compiler doesn't automatically import the type of the Vector if it isn't used elsewhere.

As a workaround, I think you can explicitly declare the Vector, like this: 

```
<fx:Vector type="starling.display.DisplayObject">
    <f:Button/>
</fx:Vector>
```

**Comment by teotigraphix** [comment]
Ok kewl. I just thought it might be something that could be fixed, if not it really doesn't matter. :)

**Comment by joshtynjala** [comment]
I had planned to fix this one, but I decided to wait until after Feathers SDK 2.2 was released. I simply haven't found the time yet.

**Comment by teotigraphix** [comment]
It's fine, I just tried the Vector and it compiled fine. Seems like it shouldn't be a priority for you right now IMHO.
